lib/unicore/mktables

```diff
@@ -9762,7 +9434,7 @@ sub display_chr {
     return "\x{25CC}$chr";
 }
 
-sub join_lines($) {
+sub join_lines {
```

Comment:
There is a shift in this sub that you could name

Reply:
indeed good idea